tests/Avalonia.RenderTests/Controls/BorderTests.cs

```diff
@@ -204,7 +204,7 @@ public BorderTests()
                     {
                         Text = "Foo",
                         Background = Brushes.Red,
-                        FontFamily = "Segoe UI",
```

Comment:
This is a breaking change. Please,  add an implicit convertion operator to the FontFamily class

Reply:
Okay that's a good idea

Reply:
Have implemented implicit conversion

Reply:
If there is no an implicit conversion (I couldn't where is it btw?) then this change is no longer necessary, please revert all the FontFamily = new FontFamily( "") changes back to FontFamily = "".

Reply:
I am not sure if I should revert everything to use string. Implicit conversion is just needed for backwards compatibility.

Reply:
My though process was that then this PR only shows the necessary changes, however i'll let you make the best judgement on this one :)